pandas/core/tools/datetimes.py

```diff
@@ -470,11 +470,26 @@ def to_datetime(arg, errors='raise', dayfirst=False, yearfirst=False,
         - If True, require an exact format match.
         - If False, allow the format to match anywhere in the target string.
 
-    unit : string, default 'ns'
-        unit of the arg (D,s,ms,us,ns) denote the unit, which is an
-        integer or float number. This will be based off the origin.
-        Example, with unit='ms' and origin='unix' (the default), this
-        would calculate the number of milliseconds to the unix epoch start.
+    unit : string, default is 'N'
+        The unit of the arg. Uses a subset of the pandas offset aliases.
+
+        - 'Y', 'A' for yearly (long term average of 365.2425 days)
```

Comment:
remove Y, A, M here I think we deprecated these anyhow

Reply:
Already my working version is a deprecation for "Y", "A", "M" and "W".  The "W" deprecation is discussed earlier in this conversation.

Reply:
so don't mention the deprecated ones